ecommerce/backend/API/templates/acc_active_email.html

```diff
@@ -0,0 +1,6 @@
+{% autoescape off %}  
+Hi {{ user.username }},  
+Please click on the link to confirm your registration,
+http://127.0.0.1:5000/verify/{{ uid }}
+If you think, it's not you, then just ignore this email.  
+{% endautoescape %}
```

Comment:
I am not sure how to handle it but the ip address needs to be the production one when in production.. maybe have it depend on an environment variable somehow?

Reply:
@bekir96 it relates you.